WickedEngine/Shaders_SPIRV.vcxproj

```diff
@@ -55,7 +55,7 @@
       <AdditionalOptions>-flegacy-macro-expansion -spirv -fvk-use-dx-layout %(AdditionalOptions)</AdditionalOptions>
     </FxCompile>
     <PreBuildEvent>
-      <Command>compile_shaders_spirv.py
+      <Command>python compile_shaders_spirv.py
```

Comment:
This doesn't let this project compile on Appveyor, would you remove the "python" please?

Reply:
Reverted, sorry